Sources/APIInterface/include/WordPressComRESTAPIInterfacing.h

```diff
@@ -0,0 +1,15 @@
+@import Foundation;
+
+@protocol WordPressComRESTAPIInterfacing
```

Comment:
Two notes on the naming:

- I used "REST" and "API", instead of "Rest" and "Api" because they are acronyms and it feels more correct to have them all uppercase (when not used as the first word of a method).
- I used "Interfacing" to align with Swift naming conventions. In that context, an object conforming to this protocol should "interface with the WordPress.com REST API" which is exactly what `WordPressComRestApi` does.

I'm a bit concerned by the ambiguity between "interfacing" as a verb and "interface" as a noun often used in programming languages to define something similar to an Objective-C or Swift protocol. Any suggestion for an alternative name is welcome.